pydexec/command.py

```diff
@@ -77,6 +77,10 @@ def env_opt(self, opt_key, env_key, default=None, required=False,
                 '"%s" for program "%s"' % (env_key, opt_key, self._program))
         return self
 
+    def _env_get(self, env_key, remove=False, default=None):
+        env_op = self._env.pop if remove else self._env.get
+        return env_op(env_key, default)
```

Comment:
This doesn't look like that big a win. Maybe we can get the `if env_val is not None: ...` bit in by accepting `opt_name=None`/`opt_name='foo'` or `pre_args=[]`/`pre_args=['--foo']` or `arg_func=self._args.append`/`arg_func=lambda a: self._args.extend(['--foo', a])` or something?